### DISCUSSION TITLE
Cant lauch Regolith Setting and problem with Spotify.

### DISCUSSION THREAD

**Discussion by betonomochalka** [regolith-linux/regolith-desktop#975] [Q&A] [answered]
I just installes regolith and i cant launch setting. And i have a problem with Log in Spotify. When i click log in it must redirect me to browser, but it takes a while to redirect (~3 minutes).

**Comment by betonomochalka** [comment]
And i have also problem with Terminal. When im trying to launch it, i takes ~3 minutes also. With super+enter or with super+space, it doesn't matter.

**Comment by nataly-novak** [comment]
Regolith control center does not have a package, it seems, not sure if this is intended behaviour. I also have the same issue with terminal - and even the one from the menu takes a while to launch. As for weird login behaviour, my nextcloud now wants me to connect it after every reboot.

**Reply by SoumyaRanjanPatnaik** [reply]
Regolith control center does have a package though. Can you try installing it using
```
sudo apt install regolith-control-center
```

**Reply by nataly-novak** [reply]
I actually did this before because I had an issue with it when it could not be upgraded when I was upgrading my install to get a version upgrade. So the result is:

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package regolith-control-center is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
 is only available from another source

E: Package 'regolith-control-center' has no installation candidate

**Comment by SoumyaRanjanPatnaik** [comment]
@betonomochalka @nataly-novak are you folks on `mantic`? If yes, can you check if you have the `xdg-desktop-portal-regolith` package installed?

**Reply by nataly-novak** [reply]
Also can't be located. I followed the upgrade instruction and the version seems to be 3.1, but there seems to be something off.

**Comment by nataly-novak** [comment, marked as answer]
Ok, so I looked into the other theme and It's probably manic vs mantic
EDIT: it was! The terminal is loading fast, the nextcloud logs in on its own, things are working.
So the solution is to check  sudo nano /etc/apt/sources.list.d/regolith.list , fix any instances of manic into mantic, and then run update/upgrade. If you had issues with control center and removed the previous one - install it manually afterwards.

**Reply by kgilmer** [reply]
Sorry about that typo @nataly-novak , it's hit a few people.  And thanks for reporting your solution :)
